src/main/java/actions/PauseResumeTrackingAction.java

```diff
@@ -13,13 +14,21 @@
 public class PauseResumeTrackingAction extends AnAction {
     private final ScreenRecorder screenRecorder = ScreenRecorder.getInstance();
 
+    private static PauseResumeTrackingAction instance = null;
+
+    public static PauseResumeTrackingAction getInstance() {
+        return instance;
+    }
+
     /**
      * Update the text of the action button. If the tracking is not started, the button is disabled.
      *
      * @param e The action event.
      */
     @Override
     public void update(@NotNull AnActionEvent e) {
+        if (instance == null)
+            instance = this;
```

Comment:
I'm guessing we're doing these checks in here instead of in getInstance (where we would create a new instance) because we don't know where this class is being instantiated (some HTML stuff). 

Wherever it is being instantiated, it's probably the default constructor. So if we write our own implementation of the default constructor we can put a debug statement in it to see if it's being called and if so, we could set `instance = this` in there. Then we can remove these checks.